Services/AccessControl/classes/class.ilObjectPermissionStatusGUI.php

```diff
@@ -458,8 +458,8 @@ function getAvailableRolesTableData()
 		{
 			$result_set[$counter]["img"] = in_array($role['obj_id'],$this->user_roles) ? self::IMG_OK : self::IMG_NOT_OK;
 
-			if(is_subclass_of($this->object->plugin, 'ilPlugin') && $role["parent"] == $this->object->getRefId()) {
-				$result_set[$counter][] = ilPlugin::lookupTxtById($this->object->getType(), ilObjRole::_removeObjectId($role["title"]));
+			if(is_subclass_of($this->object, ilObjectPlugin::class) && $role["parent"] == $this->object->getRefId()) {
```

Comment:
@shecken any objections from your side? The last changes here were made by you in a PR. LGTM

Reply:
Hi, i don't have any objections. LGTM